crates/volta-core/src/distro/package.rs

```diff
@@ -521,12 +579,13 @@ impl Installer {
                     "--loglevel=warn",
                     "--no-update-notifier",
                     "--no-audit",
+                    "--color=always",
```

Comment:
It's related to `--verbose` because we're only showing the output from this `npm` command if the user has specified `--verbose`. And in those cases, we are managing the output, so `npm` detects that it's _not_ a TTY and suppresses all colors. Then when we show it to the user, it's monotone and hard to parse.

Reply:
Then wouldn't it make sense for us to check for TTY and make that decision dynamically? If someone is calling us from a script they won't want to see the colors.

Reply:
Good point, we can do that easily :+1: